financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -263,7 +263,7 @@ async function createReferencePriceFeedForEmp(logger, web3, networker, getTime,
   });
 
   // Infer lookback from liquidation liveness.
-  if (emp) {
+  if (emp && defaultConfig) {
```

Comment:
`Object.assign` would revert if there is no defaultConfig

Reply:
Nice catch